src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.circulation.domain.policy;
+
+import io.vertx.core.json.JsonObject;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+public class OverdueFinePolicy {
+  private final String id;
+  private final String name;
```

Comment:
Do all policies have an ID and a name?

If they do, I think it is worth reducing the duplication here by extracting a super class with those fields and properties.

Reply:
Done